src/transformers/models/resnet/modeling_resnet.py

```diff
@@ -123,7 +123,14 @@ class ResNetBasicLayer(nn.Module):
     A classic ResNet's residual layer composed by two `3x3` convolutions.
     """
 
-    def __init__(self, in_channels: int, out_channels: int, stride: int = 1, activation: str = "relu"):
+    def __init__(
+        self,
+        in_channels: int,
+        out_channels: int,
+        stride: int = 1,
+        activation: str = "relu",
+        downsample_in_bottleneck: bool = False,
```

Comment:
I see the purpose of `downsample_in_bottleneck` here: it prevents one from distinguishing between `ResNetBottleNeckLayer` and `ResNetBasicLayer`. But it is not being use in this class.
So, I think it is valuable to include a docstring here, and clarify why `downsample_in_bottleneck` isn't used.

Reply:
Agree! It's cleaner.